src/displayBackend/wayland/Compositor.cpp

```diff
@@ -76,5 +74,22 @@ void Compositor::release()
 	}
 }
 
+wl_compositor* Compositor::getCompositor() const
```

Comment:
Put online getters and setters to the header. In this case, they will be automatically inlined.

Reply:
Done